specs/scoring_spec.rb

```diff
@@ -41,22 +41,28 @@
   end
 
   describe 'highest_score_from' do
-    it 'returns nil if no words were passed' do
-    end
+   it 'returns nil if no words were passed' do
+     Scrabble::Scoring.highest_score_from([]).must_be_nil
+   end
 
-    it 'returns the only word in a length-1 array' do
-    end
+   it 'returns the only word in a length-1 array' do
+     Scrabble::Scoring.highest_score_from(["snail"]).must_equal "snail"
+   end
 
-    it 'returns the highest word if there are two words' do
-    end
+   it 'returns the highest word if there are two words' do
+     Scrabble::Scoring.highest_score_from(["dog", "quail"]).must_equal "quail"
+   end
 
     it 'if tied, prefer a word with 7 letters' do
+      Scrabble::Scoring.highest_score_from(["aaaaaaa", "ggge"]).must_equal "aaaaaaa"
     end
 
     it 'if tied and no word has 7 letters, prefers the word with fewer letters' do
+      Scrabble::Scoring.highest_score_from(["aaaaaa", "ggg"]).must_equal "aaaaaa"
```

Comment:
Read the description here.  "Prefer the word with **fewer** letters"  Huh.

Should be:
```ruby
      Scrabble::Scoring.highest_score_from(["aaaaaa", "ggg"]).must_equal "ggg"
      Scrabble::Scoring.highest_score_from(["ggg", "aaaaaa"]).must_equal "ggg"
```